docsrc/imap/reference/manpages/systemcommands/deliver.rst

```diff
@@ -63,15 +63,15 @@ Options
     . If the ACL on *mailbox* does not grant the sender the "p" right,
     the delivery fails.
 
-.. option:: -a  auth-id
+.. option:: -a auth-id, --auth-id=auth-id
 
     Specify the authorization id of the sender.  Defaults to "anonymous".
 
-.. option:: -q  user-id
+.. option:: -q, --ignore-quota
```

Comment:
Was the user-id argument value just an error in the documentation?

Reply:
Yeah, the `-q` option doesn't take an argument, it just sets a flag.  `userid` is part of the "everything else after getopt has finished parsing", used by all options, rather than specific to `-q`.

I guess whoever wrote the documentation misread or misunderstood something